Tests/SonarScanner.MSBuild.PreProcessor.Test/EmbeddedAnalyzerInstallerTests.cs

```diff
@@ -68,6 +68,21 @@ public void InstallAssemblies_NullPlugins_ThrowsArgumentNullException()
             ((Action)(() => embeddedAnalyzerInstaller.InstallAssemblies(null))).Should().Throw<ArgumentNullException>().And.ParamName.Should().Be("plugins");
         }
 
+        [TestMethod]
+        public void InstallAssemblies_ValidPlugins_PluginsVersionLogInfo()
+        {
+            var localCacheDir = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext);
+            var logger = new TestLogger();
+            var requestA = new Plugin("p111", "1.0-SNAPSHOT", "p1.zip");
```

Comment:
I find the naming a bit confusing. It's not a request, it's a plugin so we can name it accordingly. The variable could also the plugin name.
```suggestion
            var pluginP1 = new Plugin("P1", "1.0-SNAPSHOT", "p1.zip");
```
and for the next var, we can do the same

Reply:
Copied from an existing test and did not think to make it more clear 😅